docs/installation/server-config/security.rst

```diff
@@ -1,4 +1,242 @@
 .. _ksql-security:
 
 Securing KSQL
--------------
\ No newline at end of file
+=============
+
+KSQL supports many of the security features of both Apache Kafka and the |sr|.
+
+- KSQL supports Apache Kafka security features such as :ref:`SSL for encryption <kafka_ssl_encryption>`,
+  :ref:`SASL for authentication <kafka_sasl_auth>`, and :ref:`ACLs <kafka_authorization>`.
+- KSQL supports :ref:`Schema Registry security features <schemaregistry_security>` such as SSL and SASL.
+
+.. contents:: Table of Contents
+    :local:
+    :depth: 1
+
+.. _steps-configure-security:
+
+Configuring Security for KSQL
+-----------------------------
+
+To configure security for KSQL, add your configuration settings to the ``<path-to-confluent>/etc/ksql/ksql-server.properties``
+file and then :ref:`start the KSQL server <install_ksql-server>` with your configuration file specified.
+
+.. code:: bash
+
+    $ <path-to-confluent>/bin/ksql-server-start <path-to-confluent>/etc/ksql/ksql-server.properties
+
+---------------------------------------
+Configuration Examples for Apache Kafka
+---------------------------------------
+
+    To connect to a secured Kafka cluster, Kafka client applications must provide their security credentials. The following
+    are common configuration examples.
+
+    **Connect to a secure Kafka cluster using PLAIN SASL and SSL, where the certificates have been signed by a CA trusted by
+    the default JVM trust store**
+
+    In this example, the following entries are added to the KSQL server configuration file (``ksql-server.properties``). This
+    configuration enables KSQL to connect to a secure Kafka cluster using PLAIN SASL, where the SSL certificates have been
+    signed by a CA trusted by the default JVM trust store. Other options include GSSAPI and Kerberos.
+
+    .. code:: bash
+
+        security.protocol=SASL_SSL
+        sasl.mechanism=PLAIN
+        sasl.jaas.config=\
+            org.apache.kafka.common.security.plain.PlainLoginModule required `
+            username="<authenticated-ksql-user>" `
+            password="<password>";
+
+    **Run KSQL against an Apache Kafka cluster in Confluent Cloud**
```

Comment:
Please make this its own section: "Using KSQL with Confluent Cloud"

Reply:
we already have a topic call "using ksql with ccloud" (http://staging-docs.confluent.io.s3-website-us-west-2.amazonaws.com/PR/587/4.1.0/ksql/docs/installation/server-config/ccloud-ksql.html)

Reply:
Maybe just a link then?

Reply:
link is there now